lib/Input.js

```diff
@@ -127,20 +127,18 @@ class Mouse {
    * @param {number} y
    * @param {!Object=} options
    */
-  async click(x, y, options) {
+  async click(x, y, options = {}) {
     this.move(x, y);
     this.down(options);
-    if (options && options.delay)
+    if (typeof options.delay === 'number')
```

Comment:
This is a slight change in functionality, if someone explicitly passes in 0. I'm not sure what the right thing to do is here, but I just thought I'd point it out.